### PR TITLE
adding method to renew the lease.

### PR DESCRIPTION
**Pull request by nikhgup**
renew lease will be used for CVS validator job.

### CODE REVIEW

File: src/Validation.Common.Job/Leases/CloudBlobLeaseService.cs [hunk 1]
```diff
@@ -69,6 +69,33 @@ public async Task<bool> ReleaseAsync(string resourceName, string leaseId, Cancel
...
+                    AccessCondition.GenerateLeaseCondition(leaseId),
+                    options: null,
+                    operationContext: null,
+                    cancellationToken);
+
+                return LeaseResult.Success(leaseId); ;
+            }
+            catch (StorageException ex) when (ex.RequestInformation?.HttpStatusCode == (int)HttpStatusCode.Conflict)
+            {
+                return LeaseResult.Failure(); ;
```
Comment (joelverhagen) [inline src/Validation.Common.Job/Leases/CloudBlobLeaseService.cs:95]:
```suggestion
                return LeaseResult.Failure();
```

File: src/Validation.Common.Job/Leases/CloudBlobLeaseService.cs [hunk 2]
```diff
@@ -69,6 +69,33 @@ public async Task<bool> ReleaseAsync(string resourceName, string leaseId, Cancel
...
+
+            try
+            {
+                await blob.RenewLeaseAsync(
+                    AccessCondition.GenerateLeaseCondition(leaseId),
+                    options: null,
+                    operationContext: null,
+                    cancellationToken);
+
+                return LeaseResult.Success(leaseId); ;
```
Comment (joelverhagen) [inline src/Validation.Common.Job/Leases/CloudBlobLeaseService.cs:91]:
```suggestion
                return LeaseResult.Success(leaseId);
```

**Review by joelverhagen — APPROVED** [review]
Code looks fine. I'm curious how it's going to be used!

**Comment by nikhgup** [comment]
@joelverhagen  It will be used in cvsSubmitterJob which will be part of cvsValidator. this job can take more then 1 min to process the servicebus message and we will be having 5 mins as lock duration for servicebus message. meanwhile we will be keep scheduling a method which will renew this lease till 5 mins if processing is still in progress.

**Review by loic-sharma — APPROVED** [review]

**Pull request merged**